src/framework/Framework.Models/ValidationExpressionsValidator.cs

```diff
@@ -0,0 +1,30 @@
+/********************************************************************************
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
```

Comment:
Please change date to 2024 since its the creation date of the file

```suggestion
 * Copyright (c) 2024 Contributors to the Eclipse Foundation
```